db/schema.rb

```diff
@@ -10,11 +10,30 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2019_12_07_120858) do
+ActiveRecord::Schema.define(version: 2020_03_06_060137) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "action_items", force: :cascade do |t|
+    t.string "title"
+    t.text "description"
+    t.boolean "is_template", default: false
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.datetime "due_date"
+  end
+
+  create_table "assignments", force: :cascade do |t|
+    t.integer "status", default: 0
```

Comment:
can we change this to 'completed', since we use status means something else in reference to participants

Reply:
Are assignments only completed/not completed? Or is there "in progress, etc..."?